LED_BUTTONS/TypeDef.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TypeDef_H_
+#define TypeDef_H_
+
+typedef unsigned char             uint8_t ;
+
+typedef unsigned short int       uint16 ;
+typedef unsigned long long       uint64_t ;
+typedef unsigned long int        uint32_t ;
+
+
+
+typedef signed char               sint8_t ;
+typedef signed short  int        sint16_t ;
+typedef signed long   int        sint32_t ;
+typedef signed long long         sint64_t ;
+
+typedef float      f32_t;
+typedef double     f64_t;
+
+#define NULL_PTR ((void*)0);
+
+
+
+
+
+
+#endif /* TypeDef_H_ */
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)